src/varlinkctl/varlinkctl.c

```diff
@@ -785,7 +785,9 @@ static int verb_call(int argc, char *argv[], void *userdata) {
                                 r = 0;
                         } else {
                                 r = sd_varlink_error_to_errno(error, reply);
-                                if (r != -EBADR)
+                                if (r == -EBADE)
+                                        log_error_errno(r, "Method call %s() failed: called without 'more' flag, but flag needs to be set", method);
```

Comment:
Please match the original SD_VARLINK_ERROR_EXPECTED_MORE instead of EBADE, as the latter might come from SD_VARLINK_ERROR_SYSTEM